frontend-react/src/utils/UserUtils.ts

```diff
@@ -1,14 +1,49 @@
 import { OktaAuth } from "@okta/okta-auth-js";
 
+let sessionBroadcastChannel: BroadcastChannel | null = null;
+
+export enum SESSION_CHANNEL {
```

Comment:
Sorry, I think this is on me, but it looks like the convention in this codebase is to use PascalCase for enums:

```js
export enum SessionChannel { ... }
```

Also, I don't think it makes sense to have the channel name and events grouped in the same enum since they're used for different things:

```js
// used to identify the channel
// (all caps used here to signify a top-level constant variable)
export const SESSION_CHANNEL_NAME = "session";

// used to identify events passed through the channel
export enum SessionEvent {
    Login = "login",
    Logout = "logout"
}
```

Reply:
👍